nodes/node_crate.lua

```diff
@@ -130,7 +135,7 @@ minetest.register_node("digtron:empty_locked_crate", {
 		meta:set_string("infotext", S("Digtron Crate") .. "\n" .. S("Owned by @1", placer:get_player_name() or ""))
 	end,
 	can_dig = function(pos,player)
-		return player_permitted(pos,player)
+		return player and not minetest.is_protected(pos, player:get_player_name()) and player_permitted(pos,player)
```

Comment:
Whitespace after comma (I see it wasn’t there, but it should be, and you edit this line anyway so that won’t harm `git blame`).